### DISCUSSION TITLE
Websocket send JSON encoded bytes

### DISCUSSION THREAD

**Discussion by kontsaki** [encode/starlette#1576] [Potential Issue] [answered]
Hello,
I think I'm misunderstanding how to send the bytes returned by `orjson.dumps`, without decoding back to utf8, over websockets.
Both `websocket.send_bytes` and `websocket.send_text` have the same behavior. 
What it the correct way to achieve this?

**Comment by Kludex** [comment, marked as answer]
You can use the following `send_json` function:
```py
from typing import Any

import orjson

from starlette.websockets import WebSocket


async def send_json(websocket: WebSocket, data: dict[str, Any]):
    text = orjson.dumps(data)
    await websocket.send({"type": "websocket.send", "text": text})
```
